### PR TITLE
Add mne-microstates

### PR DESCRIPTION
**Pull request by hoechenberger**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by hoechenberger** [comment]
@larsoner 
@wmvanvliet

Could you please confirm that you're willing to co-maintain this package? Merci 👍👍👍

**Comment by larsoner** [comment]
I'm willing to co-maintain this package

**Comment by wmvanvliet** [comment]
I too am willing to co-maintain this package.

**Comment by hoechenberger** [comment]
@conda-forge/staged-recipes, this one is ready for review. Thank you!

File: recipes/mne-microstates/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{% set name = "mne-microstates" %}
+{% set version = "0.3" %}
+{% set pypi_filename = "mne_microstates" %}
```
Comment (BastianZim) [inline recipes/mne-microstates/meta.yaml:3]: This is indeed a problem with some URLs but in that case, it's better to just hardcode it.

File: recipes/mne-microstates/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ pypi_filename }}-{{ version }}.tar.gz
```
Comment (BastianZim) [inline recipes/mne-microstates/meta.yaml:10]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mne_microstates-{{ version }}.tar.gz
```

File: recipes/mne-microstates/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  sha256: cafde4b3f8384a32ed8bc8721c0b51dcbb46326b54e9b631d6d379b02f4fdf9f
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (BastianZim) [inline recipes/mne-microstates/meta.yaml:16]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

**Review by BastianZim — CHANGES_REQUESTED** [review]

File: recipes/mne-microstates/meta.yaml (hunk 3 above)
Comment (hoechenberger) [reply recipes/mne-microstates/meta.yaml:16]: This was taken straight from the template btw 🤔

Comment (BastianZim) [reply recipes/mne-microstates/meta.yaml:16]: Ahh shoot yes I forgot to change that.

File: recipes/mne-microstates/meta.yaml (hunk 1 above)
Comment (hoechenberger) [inline recipes/mne-microstates/meta.yaml:3]:
```suggestion
```

**Comment by hoechenberger** [comment]
@BastianZim I believe I've addressed all your comments 👍

**Review by BastianZim — APPROVED** [review]

**Comment by BastianZim** [comment]
I'll need to wait for the CI and then I can merge it.

**Pull request merged**